install-vmod/install-vmod

```diff
@@ -0,0 +1,77 @@
+#/bin/sh
+
+# usage:
+#     install_vmod URL [SHA512SUM]
+# deps:
+# - varnish development files
+# - curl
+# - pkg-config
+# - nproc
+# - make
+# - gcc/clang
+# - automake
+# - libtoolize
+# - whatever dependencies the vmod needs
```

Comment:
The best usage is the one you get without having to pry the script open, see `varnishreload` for example.

Reply:
I was hoping to avoid too much argument parsing :-)

Reply:
better?

Reply:
This one was more a snarky comment than a requested change :)